src/components/WalletInfo/WalletInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+
+import classNames from 'classnames';
+
+import IconButton from '../../compositions/IconButton/IconButton';
+
+import './WalletInfo.scss';
+
+
+interface WalletInfoProps {
+  isBanner?: boolean
+  className?: string
+}
+
+const WallelInfo: FC<WalletInfoProps> = ({ isBanner, className = '' }) => {
+  const walletInfoClassName = classNames('wallet-info wallet-info--padding', {
```

Comment:
```suggestion
  const walletInfoClassName = classNames('wallet-info', {
```

Unnecessary modifier.